SFS_ir/sofa_get_head_orientations.m

```diff
@@ -1,20 +1,21 @@
-function [phi,theta] = sofa_get_head_orientations(sofa,idx)
-%SOFA_GET_HEAD_ORIENTATIONS returns phi, theta from the given SOFA data set
+function [phi,theta,r] = sofa_get_head_orientations(sofa,idx)
+%SOFA_GET_HEAD_ORIENTATIONS returns phi, theta and r from the given SOFA data set
 %
-%   Usage: [phi,theta] = sofa_get_head_orientations(sofa,[idx])
+%   Usage: [phi,theta,r] = sofa_get_head_orientations(sofa,[idx])
 %
 %   Input parameters:
-%       sofa    - impulse response data set (SOFA struct/file)
-%       idx     - index of measurement for which head orientation should be
-%                 returned (default: return all head orientations)
+%       sofa   - impulse response data set (SOFA struct/file)
+%       idx    - index of measurement for which head orientation should be
+%                returned (default: return all head orientations)
 %
 %   Output parameters:
-%       phi     - head orientations in the horizontal plane / rad
-%       theta   - head orientations in the median plane / rad
+%       phi    - head orientations in the horizontal plane / rad
+%       theta  - head orientations in the median plane / rad
+%       r      - length of head orientation vector / m
```

Comment:
"length of head orientation vector" might be misleading as you would expect it to be something like `length([phi theta])`. Maybe you change this to `r      - head orientations radius / m`?